### PR TITLE
fix: Add css to style href in link component

### PR DESCRIPTION
**Pull request by emilylim04**
## Motivations

- I noticed when migrating one of our team components to the new remix home, when I used the Link component it wasn't being styled properly. In jobber online we style our a tags but in the new home we can't rely on that anymore. 

## Changes

- Added css styling to a tag 

### Added

- Css styling to a tag in link component

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

Changes can be
[tested via Pre-release](https://github.com/GetJobber/atlantis/actions/workflows/trigger-qa-build.yml)

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

**Comment by emilylim04** [comment]
Sorry still in draft!

File: packages/components/src/Link/Link.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.link {
```
Comment (emilylim04) [inline packages/components/src/Link/Link.css:1]: @darryltec I'm not sure if Atlantis follows a class name convention? Happy to update if this is completely wrong haha.

File: packages/components/src/Link/Link.tsx [hunk 2]
```diff
@@ -20,6 +21,7 @@ export function Link({
       {...(external && { target: "_blank" })}
       {...(ariaLabel && { "aria-label": ariaLabel })}
       {...(ariaExpanded && { "aria-expanded": ariaExpanded })}
+      className={styles.link}
```
Comment (emilylim04) [inline packages/components/src/Link/Link.tsx:24]: I went with the css style route vs using the typography component so I could add the underline. But if you think I should go with the typography component I can switch it up!

File: packages/components/src/Link/Link.css (hunk 1 above)
Comment (darryltec) [reply packages/components/src/Link/Link.css:1]: no strict rule on this one, link is a good addition

File: packages/components/src/Link/Link.tsx (hunk 2 above)
Comment (darryltec) [reply packages/components/src/Link/Link.tsx:24]: this works too!

**Review by darryltec — APPROVED** [review]
:shipit:

**Pull request merged**